src/components/Indicators/LoadingErrorIndicator.tsx

```diff
@@ -1,6 +1,5 @@
 import React, { useContext } from 'react';
-import styled from 'styled-components/native';
-import PropTypes from 'prop-types';
+import { styled } from '../../styles/styledComponents';
```

Comment:
can you order these imports like we did in other places?

Reply:
done